passvault/passvault_test.go

```diff
@@ -5,33 +5,41 @@
 package passvault
 
 import (
-	"testing"
 	"os"
+	"testing"
 )
 
 func TestStaticVault(t *testing.T) {
-	err := InitFromDisk("/tmp/redoctober.json")
+	// Initial create.
```

Comment:
If this comment is important, it should be expanded out: initial creation of what? What's the lifetime of the thing being created? and so forth.